### DISCUSSION TITLE
In dialog plugin provide QSelect as an option

### DISCUSSION THREAD

**Discussion by robert-niestroj** [quasarframework/quasar#12271] [Ideas / Proposals] [answered]
Hi,
the Dialog plugin gives the possibility to pass options and select them via radios, checkboxes or toggles: 
https://v1.quasar.dev/quasar-plugins/dialog#example--radios-checkboxes-toggles

Please provide an option to select them via a QSelect as i have i case where i have 200 values. Nice to have would be possibility to search in that select in the label text and multiselect :-)

**Comment by MilosPaunovic** [comment]
Maybe you can try it this way:
[https://codepen.io/milospaunovic/pen/vYWLKwa?editors=1011](https://codepen.io/milospaunovic/pen/vYWLKwa?editors=1011)

**Comment by rstoenescu** [comment, marked as answer]
The Dialog plugin with no custom component is meant to replace the native alert(), confirm() etc.
It's not meant to be complex in any way or form. There are always going to be requests for further customization, which we will not grant (because it will be an endless process). This is where Dialog with custom components come in to play (one of its reasons, anyway).
